script_runner/app_blueprint.py

```diff
@@ -71,6 +71,7 @@ def get_config() -> dict[str, Any]:
                         }
                         for p in f.parameters
                     ],
+                    "type": "read" if f.is_readonly else "write",
```

Comment:
please make this `isReadonly`, and the type a boolean as it is less ambiguous than string